arroyo-console/src/lib/data_fetching.ts

```diff
@@ -146,7 +146,7 @@ const pingFetcher = async () => {
 
 export const usePing = () => {
   const { data, error, isLoading } = useSWR('ping', pingFetcher, {
-    refreshInterval: 1000,
+    refreshInterval: 10000000,
```

Comment:
Did you mean to commit this?

Reply:
No, I add that to stop the UI from losing state when I restart the api server